README.md

```diff
@@ -345,6 +346,48 @@ To capture data from EventSources running in the same process as EventFlow, the
 
 (*) Either providerName, or providerGuid must be specified. When both are specified, provider GUID takes precedence.
 
+#### Application Insights input
+
+*Nuget package:* [**Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelementryProcessor**](https://www.nuget.org/packages/Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelemetryProcessor/)
+
+Application Insights input is designed for the following scenario:
+1. You have an application that uses Application Insights for monitoring and diagnostics.
+2. You want to send a portion of your Application Insights telemetry to some destination other than Application Insights (e.g. Azure EventHub or Elasticsearch; the assumption is there is an EventFlow output for where the data needs to go).
+
+*Usage*
+1. Add the `EventFlowTelemetryProcessor` to your Application Insights configuration file (it goes into `TelemetryProcessors` element):
+   ```xml
+   <ApplicationInsights xmlns="http://schemas.microsoft.com/ApplicationInsights/2013/Settings" >
+     <!-- ... -->
+     <TelemetryProcessors>
+        <!-- ... -->
+        <Add Type="Microsoft.Diagnostics.EventFlow.ApplicationInsights.EventFlowTelemetryProcessor, Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelemetryProcessor" />
+        <!-- ... -->
+     </TelemetryProcessors>
+     <!-- ... -->
+   </ApplicationInsights>
+   ```
+
+2. In the EventFlow configuration make sure to include the Application Insights input. It does not take any parameters:
+   ```json
+   { "type": "ApplicationInsights" }
+   ```
+
+3. In your application code, after the EventFlow pipeline is created, find the `EventFlowTelemetryProcessor` and set its `Pipeline` property to the instance of the EventFlow pipeline:
+   ```csharp
+   using (var pipeline = DiagnosticPipelineFactory.CreatePipeline("eventFlowConfig.json"))
+   {
+       // ...
+       EventFlowTelemetryProcessor efTelemetryProcessor = TelemetryConfiguration.Active.TelemetryProcessors.OfType<EventFlowTelemetryProcessor>().First();
+       efTelemetryProcessor.Pipeline = pipeline;
```

Comment:
I thought about it when writing the code but could not find a good way to eliminate this. The problem is EventFlow does not control how the pipeline (or the Application Insights input element of the pipeline) gets created, thus it is hard to automatically cause the EventFlowTelemetryProcessor to "hook up" to it.